app/package.json

```diff
@@ -87,9 +87,11 @@
     }
   },
   "devDependencies": {
+    "@types/dotenv": "^6.1.1",
     "@typescript-eslint/eslint-plugin": "^1.13.0",
     "@typescript-eslint/parser": "^1.13.0",
     "babel-eslint": "^10.0.1",
+    "dotenv": "^8.1.0",
```

Comment:
do we really need these packages?

Reply:
I had to install these packages for the .env variable usage

Reply:
We don't need these, they come as standard with create-react scripts.

Reply:
I'm afraid you're wrong. we can use it without that packages

Reply:
I mean, the process.env.REACT... would not work until I installed that, let me have a look again!

Reply:
See how I'm using 	REACT_APP_TRINGO_API

Reply:
my bad, it is working after uninstalling them too. shall I push again?

Reply:
Yes, just push again

Reply:
Done!